.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: eFwTmwDziLPmodbfNNHxHIJOHE5aG4Rmp
```

Comment:
Oops. thank you @scw1109 ... I've removed the file also re-generated `repo_token` already ...